web/src/components/organisms/Asset/AssetList/hooks.ts

```diff
@@ -181,8 +166,12 @@ export default (projectId?: string) => {
   }, [gqlCache]);
 
   useEffect(() => {
-    const assets = (data?.assets.nodes as AssetNode[]) ?? [];
-    setAssetList(assets);
+    const assets =
+      data?.assets.nodes
+        .map(asset => asset as GQLAsset)
+        .map(convertAsset)
+        .filter(asset => !!asset) ?? [];
+    setAssetList(assets as Asset[]);
```

Comment:
```suggestion
    const assets: Asset[] =
      data?.assets.nodes
        .map(asset => asset as GQLAsset)
        .map(convertAsset)
        .filter(asset => !!asset) ?? [];
    setAssetList(assets);
```

Reply:
Where possible you should avoid type assertions!

Reply:
Although a filter is added I am still getting this error message:
```
Type '(Asset | undefined)[]' is not assignable to type 'Asset[]'.
  Type 'Asset | undefined' is not assignable to type 'Asset'.
    Type 'undefined' is not assignable to type 'Asset'.
```